### DISCUSSION TITLE
Device name part of friendly name

### DISCUSSION THREAD

**Discussion by IngmarStein** [astrandb/miele#555] [Q&A]
Currently, the device name seems to be included in the "friendly name" of entities. For example

`{{ device_attr("switch.waschmaschine_eingeschaltet", "name") }}` -> `Waschmaschine`
`{{ state_attr("switch.waschmaschine_eingeschaltet", "friendly_name") }}` -> `Waschmaschine Eingeschaltet`

This work well when you display multiple entities from different devices next to each other. However, when you group all entities of a single device, it is very repetitive:

![Bildschirmfoto 2024-09-19 um 09 36 28](https://github.com/user-attachments/assets/b075152a-f9c7-44cc-b070-c9a075287568)

Contrast that to the device page which doesn't do that:

![Bildschirmfoto 2024-09-19 um 09 38 44](https://github.com/user-attachments/assets/44ae0c30-6995-453a-a00b-45a9426950a2)

Is there a way to access the `original_name` attribute visible in the entity registry which contains just the second part (`Eingeschaltet` in the example above). `{{ state_attr("switch.waschmaschine_eingeschaltet", "original_name") }}` returns `None` unfortunately.

**Comment by astrandb** [comment]
HA is responsible for the display of names. It is not specific to this integration.
If you want to create a card with many entities without device names in the display name you can click "Zum Dashboard inzufügen" on the device page. HA will then create the name attribute without device name part.

**Comment by IngmarStein** [comment]
I see. I wasn't sure how much control over that is left to the integration.

I tried the approach to add the entities from the device card, but the result is the same. For example, these sensors:

![Bildschirmfoto 2024-09-20 um 10 56 36](https://github.com/user-attachments/assets/37ba68b7-32df-42a6-85db-551b9b91fa2f)

Turn into this:

![Bildschirmfoto 2024-09-20 um 10 56 23](https://github.com/user-attachments/assets/a898c7e1-7c9b-43fd-a5fe-6608f9a7b4a3)

I guess I'll switch from auto-entities to manually curated entity lists with custom names.

Thanks for the feedback!
